google/src/main/scala/org/broadinstitute/dsde/rawls/google/GoogleUtilities.scala

```diff
@@ -115,6 +119,12 @@ trait GoogleUtilities extends LazyLogging with Retry {
     logger.debug(GoogleRequest(request.getRequestMethod, request.buildHttpRequestUrl().toString, payload, System.currentTimeMillis() - startTime, statusCode, errorReport).toJson(GoogleRequestFormat).compactPrint)
   }
 
+  private def instrumentGoogleRequest[A](request: AbstractGoogleClientRequest[A], startTime: Long, responseOrException: Either[HttpResponseException, com.google.api.client.http.HttpResponse], isRetry: Boolean)(implicit counters: GoogleCounters[A]): Unit = {
+    val (counter, timer) = counters(request, responseOrException, isRetry)
+    counter += 1
```

Comment:
Essentially, yes. For example the `counter` name is either (depending on the value of `isRetry`):

`dev.firecloud.rawls.googleService.Buckets.httpRequestMethod.get.httpResponseStatusCode.200.request`
  - in this case we're counting a successful Buckets request

`dev.firecloud.rawls.googleService.Buckets.httpRequestMethod.get.httpResponseStatusCode.500.retry`
  - In this case we're counting a retry to a Buckets request. 

I think it's valuable to know _which_ services we're retrying most often, which is why I modeled it this way.